### PR TITLE
MILESTONE-3-REACT-CAPSTONE-MATRICS WEBAPP

### PR DESCRIPTION
**Pull request by Tobinchilongo**
### What this `PR` does:

- Created Initial setup with React create app and Linters
- Implemented basic structure for Redux store and Components, and routing
- Added countries reducer and fetching data from 'Narrativa' API
- Rendered the fetch data in the UI
- Implemented filtering data for countries
- Implemented showing detailed data for one country
- Tested the redux functions and components
- Updated README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,70 +1,69 @@
...
-### `npm test`
+## Live Demo 
+[Live Demo](https://tobinscovid-19-tracker.herokuapp.com/)
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
 
-### `npm run build`
+## Presentation Video
+[Presentation Video](https://www.loom.com/share/56da292ac7744509baab0d2bc189985b)
```
Comment (bimbolabuari) [inline README.md:22]: [Optional] As required, the video should contain an highlight about a piece of the code you find interesting or proud of making.

File: README.md [hunk 2]
```diff
@@ -1,70 +1,69 @@
...
+Feel free to check the [issues page](../../issues/).
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/deployment](https://facebook.github.io/create-react-app/docs/deployment)
+## Show your support
 
-### `npm run build` fails to minify
+Give a ⭐️ if you like this project!
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify](https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify)
+## Acknowledgments
```
Comment (bimbolabuari) [inline README.md:69]: As required, Kindly give appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the author of the original design.

**Review by bimbolabuari — CHANGES_REQUESTED** [review]
Hi @Tobinchilongo, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
-  Recorded a video presenting the features of the project.
-  The project is deployed properly, the home and detail page works as expected.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 2 above)
Comment (Tobinchilongo) [reply README.md:69]: Thank you for the feedback

**Review by bimbolabuari — APPROVED** [review]
Hi @Tobinchilongo, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**